charts/pega/README.md

```diff
@@ -823,4 +823,46 @@ certificates:
     badssl.cer: |
       "-----BEGIN CERTIFICATE-----\n<<certificate content>>\n-----END CERTIFICATE-----\n"
 
-```
\ No newline at end of file
+```
+## Clustering Service Deployment (Draft)
+
+The values.yaml section for `hazelcast` is used for configuring a Client Server form of deployment, Pega nodes act as Client for the hazelcast server nodes in this form of deployment. It is supported for Pega Platform 8.6 or later. 
+
```

Comment:
we can mention that for k8s based deployments - client/server is the default and recommended deployment. Embedded is being supported only for backward compatibility